.pre-commit-config.yaml

```diff
@@ -1,6 +1,6 @@
 repos:
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: v0.3.4
+    rev: v0.4.4
```

Comment:
Is there a reason why an update to this file is needed?

Reply:
nope. in a rush I did `commit --all`
🤦🏽‍♀️